### DISCUSSION TITLE
Lets be fork friendly - If they stay true to the license (even in future)

### DISCUSSION THREAD

**Discussion by inson1** [FossifyOrg/General-Discussion#50] [Ideas]
And lets add it to our future rules. We are fork too after all.
![brave_wVP9xPX0sz](https://github.com/FossifyOrg/General-Discussion/assets/75314629/35a27d8a-7fba-4837-8b4c-9dc7b8775299)

**Comment by naveensingh** [comment]
Anyone who wants to create a fork can create a fork. We don't need to explicitly allow it. Just don't violate the GPL.

With that said, **avoid** creating forks if you are not planning a lot of changes or additional features. Releasing the same apps with different names is only going to divide the community's resources and we will end up with lots of no-longer-maintained forks.
